node-notes-cli/app.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const command = process.argv[2];
+const textString = process.argv[3];
+const idNum = parseInt(textString);
+const newNote = process.argv[4];
+
+const read = () => {
+  fs.readFile('data.json', 'utf8', (err, jsonString) => {
+    if (err) throw err;
+    const data = JSON.parse(jsonString).notes;
+    for (const [key, value] of Object.entries(data)) {
+      console.log(`${key}: ${value}`);
+    }
+  });
+};
+
+const create = textString => {
+  fs.readFile('data.json', 'utf8', (err, jsonString) => {
+    if (err) throw err;
+    const dataObj = JSON.parse(jsonString);
+    dataObj.notes[dataObj.nextId] = textString;
+    dataObj.nextId++;
+    updateNotes(dataObj);
+  });
+};
+
+const deleteNote = indexNum => {
+  fs.readFile('data.json', 'utf8', (err, jsonString) => {
+    if (err) throw err;
+    const dataObj = JSON.parse(jsonString);
+    delete dataObj.notes[indexNum];
+    updateNotes(dataObj);
+  });
+};
+
+const changeNote = (indexNum, newNote) => {
+  fs.readFile('data.json', 'utf8', (err, jsonString) => {
+    if (err) throw err;
+    const dataObj = JSON.parse(jsonString);
+    dataObj.notes[indexNum] = newNote;
+    updateNotes(dataObj);
+  });
+};
```

Comment:
The update operation should only be able to modify existing notes, not create new ones.